java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2003,4 +2003,36 @@ void testStringTitlize() {
       assertColumnsAreEqual(expected, result);
     }
   }
+
+  @Test
+  void testNansToNulls() {
+    Float[] floats = new Float[]{1.2f, Float.POSITIVE_INFINITY,
+        Float.NEGATIVE_INFINITY, null, -Float.NaN, Float.NaN, Float.MAX_VALUE, Float.MIN_VALUE,
+        435243.2323f, Float.intBitsToFloat(0x7f800001), Float.intBitsToFloat(0x7fffffff),
+        Float.intBitsToFloat(0xff800001), Float.intBitsToFloat(0xffffffff)};
+
+    Double[] doubles = new Double[]{1.2d, Double.POSITIVE_INFINITY,
+        Double.NEGATIVE_INFINITY, null, Double.NaN, -Double.NaN, Double.MAX_VALUE,
+        Double.MIN_VALUE, 435243.2323d,
+        Double.longBitsToDouble(0x7ff0000000000001L), Double.longBitsToDouble(0x7fffffffffffffffL),
```

Comment:
If these are supposed to be special double values, let's make them constants somewhere so everyone knows what they are and can reuse them without some special magic.